### ISSUE TITLE
Controlnet only changes the generating image when Ending Control Step is set to 1.0 any other value and it will have no effect at all

### ISSUE THREAD

**Issue by CRCODE22** [Mikubill/sd-webui-controlnet#2783]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Controlnet only changes the generating image when Ending Control Step is set to 1.0 any other value and it will have no effect at all and the final result would be as if control net was disabled.

### Steps to reproduce the problem

Set Control Weight:0.55
Set Starting Control Step:0
Set Ending Control Step to 1.0
preprocessor: ip-adapter_clip_sdxl_plus_vith
model:ip-adapter-plus_sdxl_vit-h [bc449f62]
Use same seed not random
If first generation the controlnet does work the second generation and after that it will not
batch size of 1
Batch count of 1

### What should have happened?

Control net should influence the generating image every time and when you change the Ending Control Step it should affect the generating image accordingly

### Commit where the problem happens

webui: version: [v1.9.0](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/adadb4e3c7382bf3e4f7519126cd6c70f4f8557b)
controlnet: ControlNet v1.1.445

### What browsers do you use to access the UI ?

Firefox or Chrome the bug is happening on both browsers.

### Command Line Arguments

```Shell
set COMMANDLINE_ARGS= --disable-safe-unpickle --reinstall-xformers --xformers
```

### List of enabled extensions

LDSR 	built-in 	None 			
Lora 	built-in 	None 			
ScuNET 	built-in 	None 			
SwinIR 	built-in 	None 			
canvas-zoom-and-pan 	built-in 	None 			
extra-options-section 	built-in 	None 			
hypertile 	built-in 	None 			
mobile 	built-in 	None 			
postprocessing-for-training 	built-in 	None 			
prompt-bracket-checker 	built-in 	None 			
soft-inpainting 	built-in 	None 			
adetailer 	https://github.com/Bing-su/adetailer.git 	main 	1edd5888 	2024-04-17 12:57:43 	unknown
sd-webui-controlnet 	https://github.com/Mikubill/sd-webui-controlnet 	main 	b4b5bfef 	2024-04-21 22:51:20 	unknown

### Console logs

```Shell
venv "K:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.9.0
Commit hash: adadb4e3c7382bf3e4f7519126cd6c70f4f8557b
Installing xformers
Launching Web UI with arguments: --disable-safe-unpickle --reinstall-xformers --xformers
2024-04-22 01:24:45.770372: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-04-22 01:24:46.925677: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
[-] ADetailer initialized. version: 24.4.2, num models: 10
ControlNet preprocessor location: K:\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2024-04-22 01:24:59,142 - ControlNet - INFO - ControlNet v1.1.445
2024-04-22 01:24:59,449 - ControlNet - INFO - ControlNet v1.1.445
Loading weights [1170aa1935] from K:\stable-diffusion-webui\models\Stable-diffusion\SD XL\wildcardxXL_v4Rundiffusion.safetensors
Creating model from config: K:\stable-diffusion-webui\repositories\generative-models\configs\inference\sd_xl_base.yaml
2024-04-22 01:25:06,494 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7862

To create a public link, set `share=True` in `launch()`.
Startup time: 48.9s (prepare environment: 14.6s, import torch: 8.7s, import gradio: 1.5s, setup paths: 5.8s, initialize shared: 0.2s, other imports: 0.8s, list SD models: 1.6s, load scripts: 6.0s, refresh VAE: 4.2s, create ui: 4.9s, gradio launch: 0.5s).
Applying attention optimization: sdp... done.
Model loaded in 8.5s (load weights from disk: 0.3s, create model: 0.6s, apply weights to model: 6.9s, move model to device: 0.2s, calculate empty prompt: 0.3s).
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
2024-04-22 01:26:06,813 - ControlNet - INFO - unit_separate = False, style_align = False
2024-04-22 01:26:07,107 - ControlNet - INFO - Loading model: ip-adapter-plus_sdxl_vit-h [bc449f62]
2024-04-22 01:26:07,141 - ControlNet - INFO - Loaded state_dict from [K:\stable-diffusion-webui\models\ControlNet\ip-adapter-plus_sdxl_vit-h.safetensors]
2024-04-22 01:26:09,250 - ControlNet - INFO - ControlNet model ip-adapter-plus_sdxl_vit-h [bc449f62](ControlModelType.IPAdapter) loaded.
2024-04-22 01:26:09,452 - ControlNet - INFO - Using preprocessor: ip-adapter_clip_sdxl_plus_vith
2024-04-22 01:26:09,452 - ControlNet - INFO - preprocessor resolution = 1024
2024-04-22 01:26:09,517 - ControlNet - INFO - Loading CLIP model on CPU.
2024-04-22 01:26:21,786 - ControlNet - INFO - Loading CLIP model on CPU.

2024-04-22 01:26:45,910 - ControlNet - INFO - Loading CLIP model on CPU.
2024-04-22 01:26:57,897 - ControlNet - INFO - Loading CLIP model on CPU.
2024-04-22 01:27:10,061 - ControlNet - INFO - Loading CLIP model on CPU.
2024-04-22 01:27:23,152 - ControlNet - INFO - ControlNet Hooked - Time = 76.34889006614685
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:30<00:00,  1.99it/s]
Total progress: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:30<00:00,  1.97it/s]
libpng warning: iCCP: known incorrect sRGB profile████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:30<00:00,  2.08it/s]
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
2024-04-22 01:28:33,279 - ControlNet - INFO - unit_separate = False, style_align = False
2024-04-22 01:28:33,279 - ControlNet - INFO - Loading model from cache: ip-adapter-plus_sdxl_vit-h [bc449f62]
2024-04-22 01:28:33,464 - ControlNet - INFO - Using preprocessor: ip-adapter_clip_sdxl_plus_vith
2024-04-22 01:28:33,464 - ControlNet - INFO - preprocessor resolution = 1024
2024-04-22 01:28:34,160 - ControlNet - INFO - ControlNet Hooked - Time = 0.8908119201660156
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:25<00:00,  2.38it/s]
Total progress: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:25<00:00,  2.31it/s]
Total progress: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [00:25<00:00,  2.41it/s]
```

### Additional information
The second generation the bug occurs and every generation after that it is as if controlnet is disabled even tough it is enabled.
_No response_

**Comment by huchenlei** [comment]
Already fixed in A1111 dev. See https://github.com/Mikubill/sd-webui-controlnet/issues/2755

**Comment by CRCODE22** [comment]
Using git checkout dev
git pull

Solved the problems
